advanced-workflows/030-routing.md

```diff
@@ -4,6 +4,27 @@
 
 For example, `example.com` could route `$branch.demo.example.com` to the latest commit on the branch `$branch` by adding a single DNS record `CNAME *.demo demotarget.webapp.io`
 
+How to set up deployments: 
+ 
+Using [webapp.io/dashboard](/dashboard), navigate to your customs domains.
```

Comment:
customs domains?

Reply:
ops must've been merging issues I've reverted it